### DISCUSSION TITLE
Build CI using netlify for the project.

### DISCUSSION THREAD

**Discussion by Uyadav207** [praveenscience/Internship-LMS-FrontEnd#45] [Ideas]
- Use netlify for  CI of the project.
- we can also watch deployed preview while developing the website.

**Comment by Uyadav207** [comment]
#20 Discussion regarding CI can be done here.

**Reply by praveenscience** [reply]
Thanks for starting this discussion. 👍

**Reply by Uyadav207** [reply]
welcome !
